src/java.desktop/macosx/native/libawt_lwawt/java2d/metal/MTLLayer.m

```diff
@@ -80,15 +80,18 @@ - (id) initWithJavaLayer:(jobject)layer
 - (void) blitTexture {
     if (self.ctx == NULL || self.javaLayer == NULL || self.buffer == nil || self.ctx.device == nil) {
         J2dTraceLn4(J2D_TRACE_VERBOSE, "MTLLayer.blitTexture: uninitialized (mtlc=%p, javaLayer=%p, buffer=%p, devide=%p)", self.ctx, self.javaLayer, self.buffer, ctx.device);
+        [self stopDisplayLink];
         return;
     }
 
     if (self.nextDrawableCount != 0) {
+        [self stopDisplayLink];
```

Comment:
Please check invoking stopDisplayLink at this place.  If a Drawable is not available, we should return from here but recheck after 16ms. A drawable might be made available on subsequent attempts.

Stopping DisplayLink at other invalid cases makes sense.

Reply:
@aghaisas Thanks for the review. Redraw of static content is driven by setNeedsDisplay, because we stop CVDisplayLink once final blit is done. Also with preliminary testing i see that we are not hitting this condition at all after CVDisplayLink is enabled. We still have a verification task under https://bugs.openjdk.java.net/browse/JDK-8258583 . Stopping or not stopping CVDisplayLink at this place is not causing any change. I will go ahead and remove stopDisplayLink at this place, in future if we hit this code because of some boundary condition we can enable it and verify.